src/signalingprotocol/SignalingProtocol.d.ts

```diff
@@ -3224,6 +3230,12 @@ export class SdkTranscriptItem implements ISdkTranscriptItem {
     /** SdkTranscriptItem vocabularyFilterMatch. */
     public vocabularyFilterMatch: boolean;
 
+    /** SdkTranscriptItem confidence. */
+    public confidence: number;
```

Comment:
These need better descriptions somewhere. Is this 0-1, 0-100, or something else?